docx/text/paragraph.py

```diff
@@ -164,6 +164,8 @@ def image_parts(self):
         blips = [drawing.xpath(".//*[local-name() = 'blip']")[0]
                  for drawing in drawings]
 
+        docPrs = [drawing.xpath(".//*[local-name() = 'docPr']")[0]
+                  for drawing in drawings]
```

Comment:
Should these `xpath` queries be changed to `xfind` since we are expecting only one result?

Reply:
`xfind` is our platform custom made function, this looks good